linux aarch64

AntoinePrv opened the pull request:
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

AntoinePrv committed:
Add linux aarch64

AntoinePrv committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.06.14.27.19

Other tools:
- conda-build 25.11.1
- rattler-build 0.53.0
- rattler-build-conda-compat 1.4.10

AntoinePrv committed:
Increase build number

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ This recipe is using a compiler, which now requires adding a build dependence on `{{ stdlib("c") }}` as well. Note that this rule applies to each output of the recipe using a compiler.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/20008167702](https://github.com/conda-forge/conda-forge-webservices/actions/runs/20008167702). Examine the logs at this URL for more detail.</sub>

AntoinePrv committed:
Add stdlib(c)

AntoinePrv committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.06.14.27.19

Other tools:
- conda-build 25.11.1
- rattler-build 0.53.0
- rattler-build-conda-compat 1.4.10

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

AntoinePrv commented:
@jjerphan 🙏

jjerphan reviewed (approved):
Thank you, @AntoinePrv!
